draft-ietf-6man-slaac-renum-08.xml

```diff
@@ -360,18 +414,20 @@ This is to align the Router Lifetime with the recommendations in <xref target="R
 </t>
 </list>
 </t>
-
-
-
-
-
+<t>
+While allowing updates to the valid lifetime in RAs could enable an attacker to invalidate addresses by setting the valid lifetime to zero, this doesn't significantly worsen the security situation. An attacker capable of sending rogue RAs already has the power to disrupt connectivity by manipulating other parameters, such as gateway or DNS information.  Therefore, accepting a zero lifetime doesn't make the system more vulnerable than it already is in the presence of such an attacker.
```

Comment:
I'd probably:
* "doesn't" -> does not

* "accepting a zero lifetime doesn't make the system more vulnerable than it already is in the presence of such an attacker."  -> "accepting a zero lifetiem is one of the many vectors available to perform DoS attacks to local nodes"  and probably add an informative reference to rfc3756

Reply:
> "doesn't" -> does not

Fixed.

>"accepting a zero lifetime doesn't make the system more vulnerable than it already is in the presence of such an attacker." ->  -> "accepting a zero lifetiem is one of the many vectors available to perform DoS attacks to local nodes" and probably add an informative reference to rfc3756

How about:

"accepting a zero lifetime doesn't make the system more vulnerable than it already is, as invalidating the prefix is just one of the many vectors available to perform DoS attacks to onlink nodes, see RFC3756"?

Reply:
I'm fine with the change. Two notes:

1) What I was meaning was to avoid "..more vulnerable" such that people don't argue "well, this is one more vector... even if at the end of the day having an extra vector doesn't make a practical difference, since there are so many!"

2) I'd put "..to onlink nodes (see [RFC3756])" within parenthesis..but you probably meant that, anyway. :-)

Reply:
> 2. I'd put "..to onlink nodes (see [RFC3756])" within parenthesis..but you probably meant that, anyway. :-)

Yep, fixed, thanks!